_data/sites.json

```
@@ -465,6 +465,17 @@
         ]
     },
 
+    {
+        "name": "Authy",
+        "url": "https://authy.com/account/delete/",
+        "difficulty": "easy",
```

```suggestion
        "difficulty": "medium",
```
I'm suggesting it to be `medium` because it is not as simple as clicking a button, you have to fill out a form, then go to e-mail, then back...